scripts/globals/weaponskills.lua

```diff
@@ -231,7 +226,7 @@ end;
 -- params: ftp100, ftp200, ftp300, wsc_str, wsc_dex, wsc_vit, wsc_agi, wsc_int, wsc_mnd, wsc_chr,
 --         ele (ELE_FIRE), skill (SKILL_STF), includemab = true
 
-function doMagicWeaponskill(attacker, target, wsID, params, tp, primary)
+function doMagicWeaponskill(player, target, wsID, tp, primary, action, params)
```

Comment:
you can't change the name of this variable without changing the rest of the function..